schemas/scalars/datetime.ts

```diff
@@ -0,0 +1,28 @@
+import { UserInputError } from "apollo-server-core";
+import { GraphQLError, GraphQLScalarType, Kind } from "graphql";
+
+export const dateTimeScalar = new GraphQLScalarType({
+  name: "DateTime",
+  description: "Date custom scalar type",
+  serialize(value: any) {
+    return new Date(value).toISOString();
+  },
+  parseValue(value: any) {
+    return new Date(value);
+  },
+  parseLiteral(ast) {
+    if (ast.kind !== Kind.STRING) {
+      throw new GraphQLError(
+        `Query Error: Can only parse strings as datetime but got a: ${ast.kind}`
+      );
+    }
+
+    if (ast.value !== new Date(Date.parse(ast.value)).toISOString()) {
```

Comment:
There's a better way of checking this. You can refer to similar solutions here: https://thewebdev.info/2021/03/27/how-to-check-if-a-string-is-a-date-string-with-javascript

Reply:
Noted po sir. How about this one po.